addons/l10n_jo_edi/models/account_move_send.py

```diff
@@ -0,0 +1,76 @@
+from odoo import _, api, models
+
+
+class AccountMoveSend(models.AbstractModel):
+    _inherit = "account.move.send"
+
+    @api.model
+    def _l10n_jo_is_edi_applicable(self, move):
```

Comment:
I don't think this method does anything more that is useful than the l10n_jo_edi_is_needed.  Maybe we keep the to_send as a dummy state, but at least do not depend on it here.  That way, at least we avoid the confusion between applicable and needed.

Reply:
Well, the state is actually needed to prevent submitting invoices more than once. If I would not rely on the state, my condition would change to be
```python
return move.l10n_jo_edi_is_needed and (move.l10n_jo_edi_error or not move.l10n_jo_edi_xml_attachment_file)
```
which is needlessly redundant

Reply:
It is confusing